apps/server/src/modules/tool/context-external-tool/context-external-tool.module.ts

```diff
@@ -15,18 +17,19 @@ import { ToolConfigurationStatusService } from './service/tool-configuration-sta
 		SchoolExternalToolModule,
 		LoggerModule,
 		UserLicenseModule,
+		AuthorizationModule,
 	],
 	providers: [
 		ContextExternalToolService,
 		ContextExternalToolValidationService,
 		ContextExternalToolAuthorizableService,
 		ToolReferenceService,
 		ToolConfigurationStatusService,
+		ContextExternalToolRule,
```

Comment:
Ich schätze in den Modulen ist auch keine saubere Grenze zwichen api und domain Module gewahrt. Bzw. das api war bisher nicht vorhanden da der api Part (die Rule) sich im Authorization Module versteckt hatte.

Ich finde es allerdings unglücklich das jetzt im Domain Module zu platzieren. Das räumt später nie wieder einer auf und wird im schlimmsten Fall noch adaptiert.

Edit: Ich könnte schwören das wir in irgendein Team Review mal angemerkt hatten, das contextExternalToolAuthorizableService aus dem context-external-tool.module.ts raus muss.

Das zieht sich dann so durch im Tool Module.